sys/include/ubjson.h

```diff
@@ -20,12 +20,16 @@
  * @defgroup sys_ubjson  Universal Binary JSON library
  * @ingroup  sys_serialization
  * @brief    Provides a library to read and write UBJSON serialized data
+ *
+ * @deprecated  This module is deprecated and will be removed in future releases.
```

Comment:
I think this note should be enough.

Reply:
The problem is that if you look up the function in the doxygen html you may end up in the header page where that warning does not show up.

Reply:
I believe it does not really hurt to add the message on the two functions that for sure will be used.

Reply:
Every `@depracated` tag will show up will show up in the https://doc.riot-os.org/deprecated.html list. Just saying.